src/lib.rs

```diff
@@ -275,10 +545,10 @@ fn parse_export_type(input: &[u8]) -> ParseResult<Export> {
   let (input, index) = parse_var_uint(input)?;
 
   let export_type = match kind_byte {
-    0x00 => ExportType::Function,
+    0x00 => ExportType::Function(Err(ParseError::UnresolvedExportType)),
```

Comment:
Just to confirm - this is the default that will be populated later on (if it's possible)

Reply:
Yeah, it's populated later or left as-is for `skip_types: true`. Then in the CLI we can mark exports that have this as an `unknown` type until we can come up with a fix.

Reply:
Lovely